pkg/provisioner/ironic/ironic.go

```diff
@@ -1295,3 +1323,28 @@ func (p *ironicProvisioner) PowerOff() (result provisioner.Result, err error) {
 
 	return result, nil
 }
+
+// getImageChecksum parses the checksum and returns image_hash and image_value
```

Comment:
The Ironic fields are `image_hash_algo` and `image_hash_value`; shortening these (in different ways) is confusing and ambiguous.

Reply:
+1. Will update.